pkg/kf/commands/apps/integration_test.go

```diff
@@ -28,10 +28,29 @@ import (
 	. "github.com/GoogleCloudPlatform/kf/pkg/kf/testutil"
 )
 
+// TestIntegration_Buildpacks uploads the sample buildpacks and then lists
+// them. It ensures the ablity to upload and list buildpacks along with
+// preparing the cluster for the rest of the integration tests (which require
+// the buildpacks).
```

Comment:
I don't think go has a specified order for running tests in so it's not guaranteed this will execute before other tests unless it's explicitly executed. I think if other tests rely on this then there should be an `InitializeBuildpacks()` function each calls.

Reply:
I think you're right.